app/_assets/javascripts/app.js

```diff
@@ -36,26 +36,42 @@ $(function () {
     })
   }
 
-  // Cookie Policy
-  // Check for cookie
-  if (!document.cookie.split(';').filter(function (item) {
-    return item.indexOf('cookie-policy=') >= 0
-  }).length) {
-    $('.cookie-policy-container').addClass('show')
-    $('.page').addClass('page-cookie-policy')
-  }
-
   // Hide banner on "I accept" and set cookie
   $('.cookie-policy-accept').on('click', function (e) {
     e.preventDefault()
 
     $('.cookie-policy-container').removeClass('show')
     $('.page').removeClass('page-cookie-policy')
-    var CookieDate = new Date()
-    CookieDate.setFullYear(CookieDate.getFullYear() + 10)
-    document.cookie = 'cookie-policy; expires=' + CookieDate.toGMTString() + ';path=/'
+    setCookie('cookie-policy', 'agreed')
   })
 
+  function getCookie (cname) {
+    var name = cname + '='
+    var ca = document.cookie.split(';')
+    for (var i = 0; i < ca.length; i++) {
+      var c = ca[i]
+      while (c.charAt(0) === ' ') {
+        c = c.substring(1)
+      }
+      if (c.indexOf(name) === 0) {
+        return c.substring(name.length, c.length)
+      }
+    }
+    return ''
+  }
+
+  function setCookie (cname, cvalue) {
+    var CookieDate = new Date()
+    CookieDate.setFullYear(CookieDate.getFullYear() + 1)
```

Comment:
Looking good. Any reason we're limiting this cookie to one year? I see that the konghq.com cookie is set to never expire.

Reply:
I've gone back and forth with people on how long it should be set. I will most likely update Konghq to also be a year.